Inc/Effects.h

```diff
@@ -537,6 +537,9 @@ namespace DirectX
             D3D12_GPU_DESCRIPTOR_HANDLE sampler);
 
         void __cdecl SetEmissiveTexture(D3D12_GPU_DESCRIPTOR_HANDLE emissive);
```

Comment:
To be consistent with other effects, I should have used ``srvDescriptor`` for the various single-parameter texture parameters.

Also, for the ``SetAlbedoTexture`` you should take a sampler.

So probably it should be:

```
void __cdecl SetEmissiveTexture(D3D12_GPU_DESCRIPTOR_HANDLE srvDescriptor);
void __cdecl SetAlbedoTexture(D3D12_GPU_DESCRIPTOR_HANDLE srvDescriptor, D3D12_GPU_DESCRIPTOR_HANDLE samplerDescriptor);
void __cdecl SetNormalTexture(D3D12_GPU_DESCRIPTOR_HANDLE srvDescriptor);
void __cdecl SetRMATexture(D3D12_GPU_DESCRIPTOR_HANDLE srvDescriptor);
```

Should also do the same for [DirectX Tool Kit for DX11](https://github.com/microsoft/DirectXTK)'s PBREffect.